ci/fpm.toml

```diff
@@ -1,5 +1,5 @@
 name = "stdlib"
-version = "0.1.0"
+version = "VERSION"
```

Comment:
Is this to be understood as a reference to the version file? Wouldn't this complicate the version retrieval in the pretty output from fpm @LKedward is working on in https://github.com/fortran-lang/fpm/pull/622?

Reply:
No, the fpm model will see no difference between a version number specified in the manifest and a version number read from a file, since the deserializer will normalize this information. Using a version file will ensure we keep the numbers in sync everywhere.

Reply:
Okay, I've found it's mentioned here: https://github.com/fortran-lang/fpm/blob/main/manifest-reference.md#project-version